chore: metric agent status (#2528)
kyma-project/telemetry-manager: docs/user/04-metrics.md

**Symptom**:

- No metrics arrive at the backend.
- In the MetricPipeline status, the `TelemetryFlowHealthy` condition has status **GatewayAllTelemetryDataDropped** or **AgentAllTelemetryDataDropped**.

**Cause**: Incorrect backend endpoint configuration (such as using the wrong authentication credentials) or the backend is unreachable.

**Solution**:

> 1. Check the error logs for the affected Pod by calling `kubectl logs -n kyma-system {POD_NAME}`

Reviewer: ```suggestion

1. Identify the failing component.
   - If the status is `GatewayAllTelemetryDataDropped`, the problem is with the gateway.
   - If the status is `AgentAllTelemetryDataDropped`, the problem is with the agent. 

2. To check the error logs for the affected component's Pod, call `kubectl logs -n kyma-system {POD_NAME}`
```

>    - For **GatewayAllTelemetryDataDropped**, check  pods `telemetry-metric-gateway`.

Reviewer: ```suggestion
   - For the gateway, check Pod `telemetry-metric-gateway`.
```

>    - For **AgentAllTelemetryDataDropped**, check pods `telemetry-metric-agent`.

Reviewer: ```suggestion
   - For the agent, check Pod `telemetry-metric-agent`.
```
